### ISSUE TITLE
Create Toxics at startup

### ISSUE THREAD

**Issue by cbgbt** [oguzbilgener/noxious#16]
**Is your feature request related to a problem? Please describe.**
It's slightly inconvenient to configure toxics after-the-fact with the client when proxies can be configured using a config file.

**Describe the solution you'd like**
This would be a clear departure from `toxiproxy`, but I wonder if it would make sense to allow creation of toxics via configuration as well.

**Describe alternatives you've considered**
None. The status quo works well.

**Comment by cbgbt** [comment]
I think this is also possible if I switch to using `core` for my use-case, so I have yet another work around. I'll leave the issue open in case you have any interest in discussing expanding `ProxyConfig` (or some other solution) for this.

**Comment by oguzbilgener** [comment]
Yeah, I also found it inconvenient to have to re-configure the toxics every time the server starts up.

Originally I left this project at feature parity, but I'm not against new useful features like this :)

**Comment by oguzbilgener** [comment]
The config file could be JSON, YAML or anything else that we could provide quick support with using a library.
